### PR TITLE
Fix for Bug #15020 dashboard update with multiproject

### PR DESCRIPTION
**Pull request by udeeshagautam**
This PR aims to fix the issue where multiple projects are showing same data for dashboard : #15020 

Post fix this is how it looks :

FixforMultiproject

### CODE REVIEW

File: extensions/data-workspace/src/dataworkspace.d.ts [hunk 1]
```diff
@@ -67,6 +67,11 @@ declare module 'dataworkspace' {
 		 */
 		createProject(name: string, location: vscode.Uri, projectTypeId: string): Promise<vscode.Uri>;
 
+		/**
+		 * Gets the project data to be placed in the dashboard container
```
Comment (kisantia) [inline extensions/data-workspace/src/dataworkspace.d.ts:71]: add to comment that it's the project data of the projectFile

File: extensions/data-workspace/src/dialogs/projectDashboard.ts [hunk 2]
```diff
@@ -132,7 +132,7 @@ export class ProjectDashboard {
 			}).component();
 
 		const headerContainer = this.createHeader(title, location);
-		this.tableContainer = this.createTables();
+		this.tableContainer = this.createTables(location);
```
Comment (kisantia) [inline extensions/data-workspace/src/dialogs/projectDashboard.ts:135]: maybe change location to projectFile for consistency

**Review by kisantia — APPROVED** [review]
tested the changes and verified the bug is fixed!

**Comment by alanrenmsft** [comment]
you have to use this format in order to close the issue automatically once the PR is merged: this PR fixes #issuenumber

**Comment by alanrenmsft** [comment]
you have to use this format in order to close the issue automatically once the PR is merged: this PR fixes #issuenumber

**Review by alanrenmsft — APPROVED** [review]

**Comment by udeeshagautam** [comment]
> you have to use this format in order to close the issue automatically once the PR is merged: this PR fixes #issuenumber

I know but I dont want to close this the port merge happens (thats why went with long line :) )

**Pull request merged**